src/nyx/roi_cache.cpp

```diff
@@ -91,7 +91,7 @@ std::vector<StatsReal> LR::get_fvals (int fcode)
 
 void LR::initialize_fvals()
 {
-	fvals.resize ((int)Nyxus::Feature3D::_COUNT_);
+	fvals.resize ((int)Nyxus::FeatureIMQ::_COUNT_);
 	for (auto& valVec : fvals)
 		valVec.push_back(0.0);
 }

```

Comment:
I am not sure if this is correct. At a given run, we other either computing 2D or 3D or IMQ features. In that case, `Nyxus::Feature2D::_COUNT_` should not be sufficient (at the moment), since that's max number of features we are going to have at a given run?

@friskluft : Your thoughts?